src/App.js

```diff
@@ -1,26 +1,16 @@
 import React, { Component } from "react";
-import logo from "./logo.svg";
-import "./App.css";
+import FlowerThumbnail from "./components/flower-thumbnail/flower-thumbnail";
```

Comment:
Probably you should rename components to pages.  That is actually what Roman did in his discipuli project